internal/pkg/ntp/ntp.go

```diff
@@ -284,20 +294,24 @@ func (syncer *Syncer) query(ctx context.Context) (lastSyncServer string, resp *n
 		}
 	}
 
-	return lastSyncServer, resp, err
+	return
```

Comment:
unnamed returns like this are very tricky and prone to errors, especially in non-trivial functions like this one

that's why we use explicit return values. so return params names are documentation, not a way to do bare `return`